baldrick/github/github_api.py

```diff
@@ -528,7 +505,9 @@ def set_check(self, name, summary, commit_hash='head', details_url=None,
         elif commit_hash == "base":
             commit_hash = self.base_sha
 
-        completed_at = datetime.utcnow().isoformat(timespec='seconds') + 'Z'
+        tt = datetime.utcnow()
+        completed_at = tt.isoformat(timespec='seconds') + 'Z'
+        summary = insert_special_message(summary, timestamp=tt)
```

Comment:
can we add a global config option to turn this on?

Reply:
What do you mean exactly? "Turn it on" even when it is not special day? Or turn if off even when it is special day? But if you can turn it off, then it won't be special anymore! :stuck_out_tongue:

Reply:
I mean this should be a thing that each deployment of the bot can choose to do or not, so it should be in the global app config.

Reply:
I don't think that's how Easter eggs work... But I'll think about it.

Reply:
it would still be an easter egg for the end user ;) We should at least provide a way of turning it off on an app level.

Reply:
Happy now? :wink: